### ISSUE TITLE
Visualization 

### ISSUE THREAD

**Issue by Alvalens** [VictoryVicy/Py-Shortest-Path#1]
Can you add a visualization to the path so it be more readable?

**Comment by VictoryVicy** [comment]
Yes i can, coz for next step, i wanna make a visualization for my graph Dijkstra's algoritm, so you can easily understand bout this code.
